subprojects/core/src/integTest/groovy/org/gradle/api/tasks/IncrementalInputsIntegrationTest.groovy

```diff
@@ -143,6 +143,9 @@ class IncrementalInputsIntegrationTest extends AbstractIncrementalTasksIntegrati
             }
         """
         file("nonIncremental").text = "input"
+        executer.beforeExecute {
+            expectDeprecationWarning()
+        }
```

Comment:
Why not declare outputs for this task instead?

Reply:
I did not declare outputs for any of the tasks and expected the deprecation warning to show that there is no change in behaviour. I think the outputs should be added when we fail the build with Gradle 6.0. WDYT?

Reply:
To me it looks like these tests are validating some unrelated behavior. Thus it's a bug for them to rely on this now deprecated functionality of needing to declare no outputs, and should be fixed right now. Why would you leave the fix for 6.0? Or is there some functionality under test here that is related to having no outputs?

Reply:
I changed the tests so they declare outputs.